admin_manual/configuration/server/security/ransomeware-protection.rst

```diff
@@ -0,0 +1,95 @@
+
+.. _ransomware_protection_label:
+
+Protection: Data Retention and Rollback
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While Ransomware Prevention mitigates risks of a range of ransomware attacks, it is not a future-proof solution, because ransomware is becoming ever-more sophisticated.
+There are known attacks that change file extensions randomly or keep them unchanged which makes them harder to detect.
+
+Ultimately there is a consensus that only one solution can provide future-proof protection from ransomware attacks: retaining data and providing the means to roll back to a particular point in time.
+
+ownCloud Ransomware Protection will, therefore, record all changes on an ownCloud Server and allow administrators to rollback user data to a particular point in time, making use of ownCloud’s integrated Versioning and Trash bin features.
+
+Doing so allows all user data that is synchronized with the server to be rolled back to its state before the attack occurred.
+We expect that a combination of Ransomware prevention and protection should reduce risks to a minimum acceptable level.
+
+Other Elements of Ransomware Protection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+==================== ============================================== ================================================
+Name                 Command                                        Description
+==================== ============================================== ================================================
+Ransomguard Scanner  ``occ ransomguard:scan <timestamp> <user>``    A command to scan the ownCloud database for
+                                                                    anomalies in a user account. It enables an administrator to determine the point in time
+                                                                    where undesired actions happened as a
+                                                                    prerequisite for restoration.
+Ransomguard Restorer ``occ ransomguard:restore <timestamp> <user>`` A command for administrators to revert all
+                                                                    operations in a user account that occurred after
+                                                                    a certain point in time.
+==================== ============================================== ================================================
+
+.. note:: 
+   ``<timestamp>`` must be in `the Linux timestamp format`.
+
+Requirements
+~~~~~~~~~~~~
+
+#. **Ransomware Protection.** It records the move operations so that it can revert them in the case of an attack. Other operations are revertible using ownCloud's existing database tables.
+#. **ownCloud Versions App.** It is required to restore older file versions and because capabilities of Ransomware Protection depend on its configuration.
```

Comment:
"Because" does not make much sense here. The capabilities of ransomware depend on the configuration. Meaning if versioning is configured to clean up versions very fast you maybe won't have the versions available that would be needed for a rollback when e.g. a week has passed after an attack occurred.

Reply:
I stripped out the latter part of the sentence, and the one for versioning. I hope that's correct.

Reply:
"It is required ... because capabilities of ransomware protection depend on its configuration" does not make sense to me. I suggest to split into two sentences:

"Is required to restore older file versions. The capabilities of Ransomware Protection depend on its configuration regarding version retention."

Same for trash bin.

Reply:
Ok, that sounds fair enough to me.

Reply:
@pmaier1, I've made a further change. Can you review, pls?